src/main/webapp/style.css

```diff
@@ -365,6 +365,41 @@ table {
   padding-left: 35px;
 }
 
+#recommendation-list a {
```

Comment:
Since recommendation list has the same styling as history list, you could remove the CSS for one of the two and rename it with a class called something like "visualization thumbnails" and add the class to both

Reply:
I tried this but it ended up erasing the styling for the images like the border and link styling.

Reply:
Oh I guess there'd be a few other things you'd need to change, so it can be kept the way it is, thanks